features/product/managing_product_variants/checking_products_in_the_shop_while_editing_them.feature

```diff
@@ -0,0 +1,26 @@
+@managing_product_variants
+Feature: Checking products in the shop while editing them
```

Comment:
```suggestion
Feature: Checking products in the shop while editing their product variants
```

Although I still think, one feature would be enough (one in which you have 2 products created in the background, one simple, one configurable)

Reply:
I must make two because we have @managing_product_variants and @managing_product If I'd make it I'd duplicate methods thats are in two other contexts